File: experiments/functions/faas-sleep-fn/sleep/handler.go
package function

import (
	"fmt"
	"time"
)

// Handle a serverless request
func Handle(req []byte) string {
	i := 0
	out := ""

	for {
		if i == 10 {
			break
		}

		time.Sleep(1 * time.Second)

		out += fmt.Sprintf("Counter... %d\n", i)

		i++
	}

	out += fmt.Sprintf("\n\nHello, Go. You said: %s", string(req))
	return out
}

File: experiments/functions/faas-sleep-fn/sleep2/handler.py
import time


def handle(req):
    """handle a request to the function
    Args:
        req (str): request body
    """

    out = ""

    for i in range(10):
        time.sleep(1)
        out += "Count... " + str(i) + "\n"

    out += "\n\n" + str(req)
    return out
